pkg/didmethod/peer/did.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package peer
+
+import (
+	"crypto"
+	"encoding/base64"
+	"encoding/json"
+	"strings"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+	errors "golang.org/x/xerrors"
+)
+
+const (
+	//numAlgo is the algorithm for choosing a numeric basis Reference: https://openssi.github.io/peer-did-method-spec/index.html#method-specific-identifier
+	numAlgo = "1"
+	//encAlgo  is the algorithm for encoding
+	encAlgo = "1"
+)
+
+//newDid creates the peer DID. For example : did:peer:11-479cbc07c3f991725836a3aa2a581ca2029198aa420b9d99bc0e131d9f3e2cbe
+func newDid(doc *did.Doc) (string, error) {
```

Comment:
This is already being delivered in PR #56

Reply:
yes i think it is not merged thats why its showing two commits .. please ignore.